node/network/statement-distribution/src/tests.rs

```diff
@@ -1053,9 +1053,12 @@ fn receiving_large_statement_from_one_sends_to_another_and_to_candidate_backing(
 			.expect("should be signed")
 		};
 
-		let metadata =
-			protocol_v1::StatementDistributionMessage::Statement(hash_a, statement.clone().into())
-				.get_metadata();
+		let metadata = protocol_v1::StatementMetadata {
+			relay_parent: hash_a,
+			candidate_hash: statement.payload().candidate_hash(),
+			signed_by: statement.validator_index(),
+			signature: statement.signature().clone(),
+		};
```

Comment:
I'd prefer a function that does the same as `get_metadata`, rather than replicating the same code - just in the scope of the tests or, if it fits in `polkadot-primitives-test-helpers` or something akin to it for the `network` subfolder.

Reply:
Right